core/src/main/resources/defaults/blueprints/7.2.16/cdp-data-engineering-ha.bp

```diff
@@ -160,7 +156,7 @@
           },
           {
             "name": "hive_service_config_safety_valve",
-            "value": "<property><name>fs.s3a.ssl.channel.mode</name><value>openssl</value></property><property><name>hive.txn.acid.dir.cache.duration</name><value>0</value></property><property><name>hive.server2.tez.session.lifetime</name><value>30m</value></property>
+            "value": "<property><name>hive.txn.acid.dir.cache.duration</name><value>0</value></property><property><name>hive.server2.tez.session.lifetime</name><value>30m</value></property>
```

Comment:
is this ok for all the provider? AWS, AZURE ... ?

Reply:
No, Only for AWS.